mne/time_frequency/psd.py

```diff
@@ -95,7 +105,7 @@ def psd_array_welch(x, sfreq, fmin=0, fmax=np.inf, n_fft=256, n_overlap=0,
 
     Returns
     -------
-    psds : ndarray, shape (..., n_freqs) or
+    psds : ndarray, shape (..., n_freqs) or (..., n_windows, ..., n_freqs)
```

Comment:
Hm, I'd have to check - I have a recollection that when you use `psd_welch` on Raw with one channel then the output is n_channels x n_freqs, but will have to take a look at `psd_array_welch` again.